custom-namespaces.yaml

```
@@ -420,6 +420,10 @@ revenue:
       type: table_view
       tables:
         - table: mozdata.stripe.itemized_payout_reconciliation
+    ad_cost_breakdowns:
+      type: table_view
+      tables:
+        - table: fenix_derived.google_ads_campaign_cost_breakdowns_v1
```

I'm pretty sure this table needs a project qualifier
```suggestion
        - table: moz-fx-data-shared-prod.fenix_derived.google_ads_campaign_cost_breakdowns_v1
```